utils/unittest/entity_test.go

```diff
@@ -0,0 +1,65 @@
+package unittest
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type IntList []uint32
+
+func (l IntList) ID() flow.Identifier {
+	return flow.MakeID(l)
+}
+
+type EmbeddedStruct struct {
+	Identities flow.IdentitySkeletonList
+	QC         *flow.QuorumCertificate
+}
+
+func (e *EmbeddedStruct) ID() flow.Identifier {
+	return flow.MakeID(e)
+}
+
+type StructWithUnsupportedFlowField struct {
+	Field chan struct{}
+}
+
+func (e *StructWithUnsupportedFlowField) ID() flow.Identifier {
+	return flow.MakeID(e)
+}
+
+// TestRequireEntityNonMalleable tests the RequireEntityNonMalleable function with different types of entities ensuring
```

Comment:
It would be good to add a case of a malleable structure where we define the ID() in a faulty way, to validate the `RequireEntityNonMalleable` behaviour.

Reply:
https://github.com/onflow/flow-go/pull/7069/files#diff-3f987212c55e79379228787d446830f2948a6e8bf0ce6fdbb405c4934ce5f733R90

Reply:
Thanks!